.env

```
@@ -4,10 +4,17 @@ EXPO_PUBLIC_VIBECODE_GROK_API_KEY=xai-ahDi8ofei1Em2chaichoac2Beehi8thu-n0tr3al
 EXPO_PUBLIC_VIBECODE_GOOGLE_API_KEY=UeHoh2oot2IWe6ooW4Oofahd6waebeiw-n0tr3al
 EXPO_PUBLIC_VIBECODE_ELEVENLABS_API_KEY=elevenlabs-api-key-oa9Shahx4Zi4oof2bei5kee9nee7eeng-n0tr3al
 
-# Firebase Configuration for LockerRoom Project
+# Firebase Configuration for LockerRoom Project (Legacy - will be removed after migration)
 EXPO_PUBLIC_FIREBASE_API_KEY=AIzaSyDcdGoo6Z2uHXNmFyXnKdltwrMUXPcp_4A
 EXPO_PUBLIC_FIREBASE_AUTH_DOMAIN=locker-room-talk-app.firebaseapp.com
 EXPO_PUBLIC_FIREBASE_PROJECT_ID=locker-room-talk-app
 EXPO_PUBLIC_FIREBASE_STORAGE_BUCKET=locker-room-talk-app.firebasestorage.app
 EXPO_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=514288923681
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_FIREBASE_MESSAGING_SENDER_ID key should go before the EXPO_PUBLIC_FIREBASE_PROJECT_ID key

(UnorderedKey)